src/PSRule.Rules.Azure/rules/Azure.SQL.Rule.ps1

```diff
@@ -163,6 +163,50 @@ Rule 'Azure.SQL.FGName' -Ref 'AZR-000193' -Type 'Microsoft.Sql/servers/failoverG
 
 #endregion Failover group
 
+#region Manintenance window
+
+# Synopsis: Configure a customer-controlled maintenance window for Azure SQL databases.
+Rule 'Azure.SQL.MaintenanceWindow' -Ref 'AZR-000440' -Type 'Microsoft.Sql/servers', 'Microsoft.Sql/servers/databases', 'Microsoft.Sql/servers/elasticPools' -Tag @{ release = 'GA'; ruleSet = '2024_09'; 'Azure.WAF/pillar' = 'Reliability'; } {
```

Comment:
Let's exclude if single Database SKU (`sku.name`) is `B`, `S0`, `S1`, `GP_Fsv2_*`, and `HS_DC_*`. OR if the database is `master`.

Reply:
Makes sense, but what is the reason for excluding the `master` database? I am curious.

Reply:
Master database is created by the system. It's applicable for the in-flight case. Most configurations can't be applied to this DB.

Reply:
@BernieWhite My question is more like, why do we have to exclude the `master` database from having a window configured? Is it because it's simply not possible to have a configuration for this? I am aware of that the database is created when deploying the logical server.

Reply:
Correct. The normal Azure options don't apply to this database directly.

Reply:
Thanks, then it's all good @BernieWhite!